cluster/manifests/skipper/daemonset.yaml

```diff
@@ -63,7 +64,7 @@ spec:
           - "-max-audit-body=0"
           - "-oauth2-tokeninfo-url={{ .ConfigItems.tokeninfo_url }}"
           - "-histogram-metric-buckets=.01,.025,.05,.075,.1,.2,.3,.4,.5,.75,1,2,3,4,5,7,10,15,20,30,60,120,300,600"
-          - "-opentracing=lightstep component-name=skipper-ingress token=$(LIGHTSTEP_TOKEN) collector=tracing.zmon.zalan.do:8444 cmd-line=skipper-ingress max-buffered-spans=4096"
+          # - "-opentracing=lightstep component-name=skipper-ingress token=$(LIGHTSTEP_TOKEN) collector=tracing.zmon.zalan.do:8444 cmd-line=skipper-ingress max-buffered-spans=4096"
```

Comment:
Note: this should not be committed :)

Reply:
That's why it's tagged 'do-not-merge'. But this is required for testing, and you wanted a PR with the current state of everything so other people can look at it.

Reply:
@dionb please bring this PR into a state where it could be merged so it can be review by your peers. Otherwise it is very hard for us to give fast feedback and ensure short cycles.

Reply:
A good practice is also to label it/ call it WIP. For me this is especially important when I go into planning with the other teams.